xla/stream_executor/gpu/BUILD

```diff
@@ -590,6 +590,33 @@ xla_test(
     ],
 )
 
+xla_test(
+    name = "gpu_cudamallocasync_allocator_test",
+    srcs = if_gpu_is_configured(["gpu_cudamallocasync_allocator_test.cc"]),
+    backends = ["gpu_any"],
+    local_defines = if_cuda_is_configured(["GOOGLE_CUDA=1"]),
+    deps = [
+        "//xla:types",
+        "//xla/service:platform_util",
+        "//xla/stream_executor",
+        "//xla/stream_executor:platform",
+        "//xla/stream_executor:platform_manager",
+        "//xla/stream_executor/gpu:gpu_stream",
+        "@com_google_absl//absl/status",
+        "@com_google_absl//absl/strings",
+        "@tsl//tsl/lib/core:status_test_util",
+        "@tsl//tsl/platform:status",
+        "@tsl//tsl/platform:statusor",
+        "@tsl//tsl/platform:test",
+        "@tsl//tsl/platform:test_benchmark",
+        "@tsl//tsl/framework:device_id",
+        "@tsl//tsl/platform:test_main",
+    ] + if_cuda_is_configured([
+        "//xla/stream_executor/cuda:cuda_platform",
+        ":gpu_cudamallocasync_allocator_header",
```

Comment:
I think needs "@local_config_cuda//cuda:cuda_headers" inside the if_cuda_is_configured